ru-RU/basis.event.md

```diff
@@ -11,9 +11,10 @@
 Инициация события осуществляется вызовом специального метода. Такие методы принято называть с префиксом `emit_`, а функция метода создается с помощью функции `basis.event.create`. (см. Событийные методы)
 
 ```js
-var Example = basis.event.Emmiter.subclass({
-  emit_myEvent: basis.event.create('myEvent') // создание события myEvent
-});
+var Events = basis.require('basis.event'),
```

Comment:
Привычка. Поправим )
23 мая 2016 г. 3:12 PM пользователь "Roman Dvornov" <
notifications@github.com> написал:

> In ru-RU/basis.event.md
> <https://github.com/basisjs/articles/pull/21#discussion_r64211532>:
>
> > @@ -11,9 +11,10 @@
> >  Инициация события осуществляется вызовом специального метода. Такие методы принято называть с префиксом `emit_`, а функция метода создается с помощью функции `basis.event.create`. (см. Событийные методы)
> >
> >  ```js
> > -var Example = basis.event.Emmiter.subclass({
> > -  emit_myEvent: basis.event.create('myEvent') // создание события myEvent
> > -});
> > +var Events = basis.require('basis.event'),
>
> Не стоит делать multiple var declaration. Один var - одна переменная.
> Именовать с большой буквы импорт не принято, только имена классов
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/basisjs/articles/pull/21/files/ec42d68f9e59483a8e3252943440a6a6a111f8a4#r64211532>
>